### ISSUE TITLE
Immutable state in PedigreeSVG

### ISSUE THREAD

**Issue by martijnvermaat** [lumc-nested/nested-editor#5]
(Copied from !2)

I currently left this unchanged, but I think it would be really nice to have the LayoutEngine return something like this:

- Member positions: `Immutable.Map<String, Immutable.List>`
- Nest positions: `Immutable.Map<Immutable.List<String>, Immutable.List>`

Where the maps are indexed as proposed above for the pedigree and the values are the calculated coordinates. This can be combined with `props.pedigree` in PedigreeSVG to do the actual drawing.

Should perhaps wait on #1

**Comment by martijnvermaat** [comment]
mentioned in merge request !2

**Comment by martijnvermaat** [comment]
Added ~172 label

**Comment by martijnvermaat** [comment]
mentioned in merge request !21

**Comment by martijnvermaat** [comment]
Status changed to closed by commit 1a08687de2a8cfaab81ff383890b7c584e4d4e95
